### PR TITLE
FPGA: upgrade OPAE to v.1.5.0-2

### PR DESCRIPTION
**Pull request by bart0sh**
Upgraded OPAE to the latest release
Tested opae-nlb-demo image on the host with Arria10 hardware

Fixes: #383

### CODE REVIEW

File: demo/opae-nlb-demo/Dockerfile [hunk 1]
```diff
@@ -13,24 +13,21 @@ FROM ${CLEAR_LINUX_BASE} as builder
...
-ARG OPAE_RELEASE=1.4.0-1
+ARG OPAE_RELEASE=1.5.0-2
 
 # workaround for a swupd failure discussed in https://github.com/clearlinux/distribution/issues/831
 RUN ldconfig
 RUN mkdir -p /usr/src/opae && \
     cd /usr/src/opae && \
-    wget https://github.com/OPAE/opae-sdk/archive/${OPAE_RELEASE}.tar.gz && \
-    tar xf *.tar.gz
+    wget -q https://github.com/OPAE/opae-sdk/archive/1.5.0-2.tar.gz -O- | tar -zx
```
Comment (mythi) [inline demo/opae-nlb-demo/Dockerfile:24]: keep `${OPAE_RELEASE} `

Comment (bart0sh) [reply demo/opae-nlb-demo/Dockerfile]: done

**Review by rojkov — APPROVED** [review]

**Review by mythi — APPROVED** [review]

**Comment by bart0sh** [comment]
@mythi @rojkov would you mind to merge this?

File: demo/opae-nlb-demo/Dockerfile [hunk 2]
```diff
@@ -13,24 +13,21 @@ FROM ${CLEAR_LINUX_BASE} as builder
 ARG CLEAR_LINUX_VERSION=
 
 RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
-    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc doxygen patch
+    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc devpkg-tbb git
```
Comment (kad) [inline demo/opae-nlb-demo/Dockerfile:16]: What is the reason to install devpkg-tbb ? Isn't it way bigger than actually needed "doxygen"?

File: demo/opae-nlb-demo/Dockerfile [hunk 3]
```diff
@@ -13,24 +13,21 @@ FROM ${CLEAR_LINUX_BASE} as builder
...
 RUN mkdir -p /usr/src/opae && \
     cd /usr/src/opae && \
-    wget https://github.com/OPAE/opae-sdk/archive/${OPAE_RELEASE}.tar.gz && \
-    tar xf *.tar.gz
+    wget -q https://github.com/OPAE/opae-sdk/archive/${OPAE_RELEASE}.tar.gz -O- | tar -zx
 
 # Build OPAE
-ADD 0001-OPAE-in-containers-don-t-enumerate-missing-device.patch /usr/src/opae/opae-sdk-${OPAE_RELEASE}
 RUN cd /usr/src/opae/opae-sdk-${OPAE_RELEASE} && \
-    patch -p1 < 0001-OPAE-in-containers-don-t-enumerate-missing-device.patch && \
```
Comment (kad) [inline demo/opae-nlb-demo/Dockerfile:30]: If patch is not needed anymore, delete it?

File: demo/opae-nlb-demo/Dockerfile (hunk 2 above)
Comment (bart0sh) [reply demo/opae-nlb-demo/Dockerfile:16]: It's a build dependency. Build fails without it with this error:
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
TBB_LIBRARIES
```

Why do we need doxygen if all what we need is nlb0 and nlb3 binaries and their runtime dependencies?
How doxygen is related to having or not having libtbb build dependency?

File: demo/opae-nlb-demo/Dockerfile (hunk 3 above)
Comment (bart0sh) [reply demo/opae-nlb-demo/Dockerfile:30]: done

File: demo/opae-nlb-demo/Dockerfile (hunk 2 above)
Comment (kad) [reply demo/opae-nlb-demo/Dockerfile:16]: Doxygen was previously a build dependency even for nlb0/3. If we don't need it anymore, great.
If tbb is new build dependency, also fine.

**Review by kad — APPROVED** [review]

**Review by mythi — APPROVED** [review]

**Pull request merged**